modules/migrations/migrate.go

```diff
@@ -250,3 +282,23 @@ func migrateRepository(downloader base.Downloader, uploader base.Uploader, opts
 
 	return nil
 }
+
+var (
+	whitelist *matchlist.Matchlist
+	blacklist *matchlist.Matchlist
+)
+
+// Init migrations service
+func Init() error {
+	var err error
+	whitelist, err = matchlist.NewMatchlist(setting.Migration.WhitelistedDomains...)
+	if err != nil {
+		return fmt.Errorf("Init migration whitelist domains failed: %v", err)
```

Comment:
This error will show up when the instance starts, right? I mean, the admin needs to see the errors as soon as possible.

Reply:
Yes. It will display when gitea start.